explorer-backend/fly/storage/repository.go

```diff
@@ -253,8 +254,8 @@ func (s *Repository) upsertTokenTransfer(ctx context.Context, v *vaa.VAA, tokenT
 	return err
 }
 
-func (s *Repository) getToken(ctx context.Context, tokenAddress string) (*TokenUpdate, error) {
-	filter := bson.D{{Key: "tokenAddress", Value: tokenAddress}}
+func (s *Repository) getToken(ctx context.Context, tokenAddress string, tokenChain vaa.ChainID) (*TokenUpdate, error) {
+	filter := bson.D{{Key: "_id", Value: getTokenId(tokenAddress, tokenChain)}}
```

Comment:
Q: `_id` seems to be the built-in id for Mongo with timestamps, a bit of randomness and an incremental counter. Do we want to replace that with our own id or could it be better to create a `tokenId` field?

Reply:
Did some research, seems to be fine to have custom value in our case since it is always deterministic and unique.

Reply:
Yes, in this case, the `_id` is the `tokenAddress:tokenChain`, which is unique.